_config.yml

```diff
@@ -153,7 +162,7 @@ content:
           fr: "240 secondes"
       DIF:
         name:
-          en: "Difficulty:"
+          en: "NIST secure hashing algorithm KangarooTwelve"
```

Comment:
maybe we should leave out the name of the algorithm (i.e. `KangarooTwelve`) completely. when/if we ever get a technical overview, i think it would be better placed there.

Reply:
I would but `NIST secure hashing algorithm` seems  a little too vague as every "SHA-N" is NIST https://en.wikipedia.org/wiki/Secure_Hash_Algorithms

Do you have any other recommendations or ways of expressing this?

Reply:
we are already pretty vague with (some of) the other bullet points, but maybe we should let it there until we have a full technical overview.

Reply:
I like that idea

Reply:
What do you feel about the latest commit? or  "Built on Keccak Team's KangarooTwelve SHA-3 variant"

Reply:
maybe `NIST approved SHA-3 variant (KangarooTwelve)`

Reply:
It doesn't say anything about what that is used for or how it relates to aeon.  that is why it would be good to have a verb somewhere or mention pow

Reply:
maybe a short version of your current commit `SHA-3 proof-of-work (KangarooTwelve)`?